custom/openclinica/const.py

```diff
@@ -1,3 +1,6 @@
+# Only for the KEMRI study. This whole file is dropped for generic study apps
+STUDY_APPS = ('Captopril Study',)
```

Comment:
That would be better. 

As soon as KEMRI has exported their data to OpenClinica (i.e. this week), we'll be able to merge the generic OpenClinica changes, and all this KEMRI-specific stuff will go.